### PR TITLE
[nodejs] enable tags test for node

### PR DESCRIPTION
**Pull request by ida613**
associated [dd-trace-js PR](https://github.com/DataDog/dd-trace-js/pull/4482)

## Workflow

1. ⚠️ Create your PR as draft ⚠️
2. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
3. Mark it as ready for review
    * Test logic is modified? -> Get a review from RFC owner. We're working on refining the `codeowners` file quickly.
    * Framework is modified, or non obvious usage of it -> get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

:rocket: Once your PR is reviewed, you can merge it!

🛟 [#apm-shared-testing](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X) 🛟

## Reviewer checklist

* [ ] If PR title starts with `[<language>]`, double-check that only `<language>` is impacted by the change
* [ ] No system-tests internal is modified. Otherwise, I have the approval from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)
* [ ] CI is green, or failing jobs are not related to this change (and you are 100% sure about this statement)
* [ ] A docker base image is modified?
    * [ ] the relevant `build-XXX-image` label is present
* [ ] A scenario is added (or removed)?
    * [ ] Get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

### CODE REVIEW

**Review by Qard — APPROVED** [review]

**Review by robertomonteromiguel — CHANGES_REQUESTED** [review]
There are test failures related with this change

File: utils/build/docker/nodejs/parametric/server.js [hunk 1]
```diff
@@ -80,13 +80,14 @@ app.post('/trace/span/start', (req, res) => {
   const extracted = tracer.extract('http_headers', convertedHeaders)
   if (extracted !== null) parent = extracted
 
+  const tags = { service: request.service }
+  for (const [key, value] of request.span_tags) tags[key] = value
```
Comment (simon-id) [inline utils/build/docker/nodejs/parametric/server.js:84]:
```suggestion
  Object.assign(tags, request.span_tags)
```

File: utils/build/docker/nodejs/parametric/server.js [hunk 2]
```diff
@@ -80,13 +80,14 @@ app.post('/trace/span/start', (req, res) => {
...
+
   const span = tracer.startSpan(request.name, {
     type: request.type,
     resource: request.resource,
     childOf: parent,
-    tags: {
-        service: request.service
-    }
+    tags: tags
```
Comment (simon-id) [inline utils/build/docker/nodejs/parametric/server.js:90]:
```suggestion
    tags
```

File: tests/parametric/test_dynamic_configuration.py [hunk 3]
```diff
@@ -692,7 +692,6 @@ def test_trace_sampling_rules_override_rate(self, library_env, test_agent, test_
     )
     @bug(context.library == "ruby", reason="RC_SAMPLING_TAGS_RULE_RATE is not respected")
     @bug(context.library <= "dotnet@2.53.2", reason="Applies rate from local sampling rule when no remote rules match.")
-    @missing_feature(library="nodejs")
```
Comment (simon-id) [inline tests/parametric/test_dynamic_configuration.py:695]: please use the version filters like the other PRs

Comment (simon-id) [reply tests/parametric/test_dynamic_configuration.py:695]: refresher:
```python
@missing_feature(context.library < "nodejs@x.x.x")
```

**Review by robertomonteromiguel — APPROVED** [review]

**Pull request merged**